tests/framework/session/test_session.py

```diff
@@ -920,3 +923,25 @@ def test_setup_logging_using_omega_config_loader_class(
     ).as_posix()
     actual_log_filepath = call_args["handlers"]["info_file_handler"]["filename"]
     assert actual_log_filepath == expected_log_filepath
+
+
+@pytest.mark.parametrize("params", ["a=1,b.c=2", "a=1,b=2,c=3"])
```

Comment:
```suggestion
@pytest.mark.parametrize("params", ["a=1,b.c=2", "a=1,b=2,c=3", ""])
```
Would be good to test the case that no params are passed here also - does adding it as a test case work?

Reply:
Yeah this would work, added it to the test!